lib/ci-stack.ts

```diff
@@ -202,6 +203,10 @@ export class CIStack extends Stack {
       accessLogBucket: auditloggingS3Bucket.bucket,
     });
 
+    const waf = new JenkinsWAF(this, {
+      loadBalancer: externalLoadBalancer.loadBalancer,
```

Comment:
So this pointed to the Jenkins ALB? And will dynamically adjust if the ALB get replaced?

Reply:
Yes!